utils/src/main/scala/com/normation/utils/Control.scala

```diff
@@ -74,10 +74,15 @@ object Control {
   def sequencePar[U,T](seq:ParSeq[U])(f:U => Box[T]) : Box[Seq[T]] = {
     val buf = scala.collection.mutable.Buffer[T]()
     @volatile var stop = Option.empty[EmptyBox]
-    seq.foreach { u => if(stop.isEmpty) f(u) match {
-      case Full(x) => buf += x
-      case e:EmptyBox => stop = Some(e)
-    } }
+    seq.foreach { u =>
+      if(stop.isEmpty) { f(u) match {
+        case Full(x) =>
+          buf.synchronized{
```

Comment:
@ncharles: we also want to limit the number of translation between collection. So we don't want to use a synchronised java collection and then have to change it back to a scala seq